utils/build-presets.ini

```diff
@@ -2365,6 +2365,29 @@ mixin-preset=stdlib_DA_standalone,build
 test
 validation-test
 
+[preset: stdlib_MSR_standalone_minimal,build]
```

Comment:
Can you put a comment here explaining what MSR is? Also, Can you change it to MR? Generally each letter stands for a different category. I would even consider S for size. Your thoughts?

Reply:
> I would even consider S for size. Your thoughts?
Do you mean just "S"? That actually sounds the best to me.

Reply:
Yes